content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -19,17 +19,47 @@ We are continuously testing the Data Proxy with Cloudflare Workers. [Open an iss
 
 </TopBlock>
 
-
 ## Prerequisites
 
-- Free [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) database with an accessible URL
-  - e.g. `mongodb+srv://<user>:<password>@cluster.mongodb.net`.
-- Free [Cloudflare Worker](https://workers.cloudflare.com/) account.
-- Free [GitHub](https://github.com/) account.
-- Node.js & NPM installed.
-- Git installed.
+- Free [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) account
+- Free [Cloudflare Workers](https://workers.cloudflare.com/) account
+- Free [GitHub](https://github.com/) account
+- Node.js & NPM installed
+- Git installed
+
+## 1. Get a MongoDB Atlas connection string
+
+After you create a MongoDB Atlas database cluster, you can get its connection string. By default, that connection string does not include a database name.
+
+When you import your project in the Prisma Data Platform at [step 8](#8-import-your-project-into-the-prisma-data-platform), you must provide a connection string that includes a database name.
+
+You do not need to manually create the database in MongoDB Atlas even if it does not yet exist. The project creation workflow in the Prisma Data Platform will create the database.
+
+1. [Create a free shared cluster](https://www.mongodb.com/docs/atlas/tutorial/create-new-cluster/) in MongoDB Atlas.
+2. On the _Security Quickstart_ screen, complete the initial security configuration.  
+   1. Add a new user and save the password you assign to it. 
+      <Admonition>
+
+      **Note**<br /><br />
 
-## 1. Set up your Application
+      The user is later added to the connection string which does not include the password.
```

Comment:
This sentence needs to be clarified... does it mean, in relation to Step 1, that: "The user you define in the step is later added to the connection string, which does not include the password."

The bit about the password is confusing... do we need to mention the password there?

Reply:
Hm, I understand. I've shortened this to just mention that the goal is to add a new database user without clarifying details about the connection string, users, or password.